### ISSUE TITLE
Complete or near-complete absence of specific antibody response to pneumococcus vaccine

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5610]
**HPO term**
https://hpo.jax.org/app/browse/term/HP:0410300

**Suggested revision of textual definition**
Complete or near-complete absence of specific antibody response to unconjugated pneumococcus vaccine
unconjugated

**Comment by pnrobinson** [comment]
Same for Partial absence of specific antibody response to (unconjugated) pneumococcus vaccine

add unconjugated

**Comment by nicolevasilevsky** [comment]
I revised the labels to be:
Complete or near-complete absence of specific antibody response to unconjugated pneumococcus vaccine

Partial absence of specific antibody response to unconjugated pneumococcus vaccine

@pnrobinson is this the correct course of action?
